frontera/contrib/backends/hbase.py

```diff
@@ -133,27 +134,27 @@ def get_interval(score, resolution):
         random_str = int(time() * 1E+6)
         data = dict()
         for request, score in batch:
-            domain = request.meta['domain']
-            fingerprint = request.meta['fingerprint']
+            domain = request.meta[b'domain']
+            fingerprint = request.meta[b'fingerprint']
             if type(domain) == dict:
-                partition_id = self.partitioner.partition(domain['name'], self.partitions)
-                host_crc32 = get_crc32(domain['name'])
+                partition_id = self.partitioner.partition(domain[b'name'], self.partitions)
+                host_crc32 = get_crc32(domain[b'name'])
             elif type(domain) == int:
                 partition_id = self.partitioner.partition_by_hash(domain, self.partitions)
                 host_crc32 = domain
             else:
                 raise TypeError("domain of unknown type.")
             item = (unhexlify(fingerprint), host_crc32, request.url, score)
             score = 1 - score  # because of lexicographical sort in HBase
-            rk = "%d_%s_%d" % (partition_id, "%0.2f_%0.2f" % get_interval(score, 0.01), random_str)
+            rk = b"%d_%s_%d" % (partition_id, b"%0.2f_%0.2f" % get_interval(score, 0.01), random_str)
```

Comment:
This won't work for Python 3 versions less than 3.5. http://legacy.python.org/dev/peps/pep-0461/
I would suggest forming a normal string and then applying `to_bytes`. This needs to be changed in other parts of the PR as well.